spec/system/casa_admins/edit_spec.rb

```diff
@@ -22,13 +24,21 @@
       expect(page).to have_text "New admin created successfully"
       expect(admin.email).to eq expected_email
       expect(admin.display_name).to eq expected_display_name
+      expect(admin.phone_number).to eq expected_phone_number
     end
   end
 
   context "with invalid data" do
-    it "shows error message for empty email" do
+    let(:role) { "admin" }
+    before do
       visit edit_casa_admin_path(admin)
+      fill_in "Email", with: "newemail@example.com"
+      fill_in "Display name", with: "Kadehara Kazuha"
```

Comment:
seems fine https://genshin-impact.fandom.com/wiki/Kaedehara_Kazuha

Reply:
🥳 🥳  guess you found my easter eggs